app/main/views/services.py

```diff
@@ -41,6 +43,10 @@ def list_services():
 
     services = Service.query.filter(
         Service.framework.has(Framework.expired == false())
+    ).order_by(
+        asc(Service.framework_id),
```

Comment:
The framework_id is an internal database ID - not guaranteed to be in any particular order, it will depend how the migrations ran and what mood Postgres was in at the time.  (I know that in fact it will probably give us the order we want, but I don't think we should rely on it.)
Below is how my Frameworks table looks, but I'm sure I looked in the preview (or maybe prod) DB and the IDs were completely different - I think one framework had ID 60 or something.
```
digitalmarketplace=# select * from frameworks;
 id |   name    | expired
----+-----------+---------
  1 | G-Cloud 6 | f
  2 | G-Cloud 4 | t
  3 | G-Cloud 5 | f
(3 rows)
```